### ISSUE TITLE
Cannot update/install libthemis on Centos/Rocky Linux 8.6 with yum

### ISSUE THREAD

**Issue by dev10** [cossacklabs/themis#922]
**Describe the bug**
I successfully installed `libthemis-devel` on Rocky Linux 8.5 (based on RHEL 8) on Thursday May 12 2022 with `sudo yum install libthemis-devel`.
This is as described here: https://docs.cossacklabs.com/themis/installation/installation-from-packages/#centos-rhel-oracle-linux
Since Tuesday May 31 2022 when trying to run `sudo yum check-update` it fails because of 404 not found.

I had to disable it with `sudo yum-config-manager --disable cossacklabs` until the repository is repaired.

When looking at the repo it does seem the xml file `dbae36491ad0ee21f407dfa338160598684f14fdfc1dfd62aa498f1ac7066e90-filelists.xml.gz` is missing and is referred to by `https://pkgs-ce.cossacklabs.com/stable/centos/8/x86_64/repodata/repomd.xml` 

**To Reproduce**
1. Using the terminal on Rocky Linux 8.5
2. Run `sudo yum check-update` or `sudo yum update`
3. See the following error:
```
Cossack Labs stable - x86_64                                                                                                                                                       838  B/s | 3.1 kB     00:03    
Errors during downloading metadata for repository 'cossacklabs':
  - Status code: 404 for https://pkgs-ce.cossacklabs.com/stable/centos/8/x86_64/repodata/dbae36491ad0ee21f407dfa338160598684f14fdfc1dfd62aa498f1ac7066e90-filelists.xml.gz (IP: 178.63.6.188)
Error: Failed to download metadata for repo 'cossacklabs': Yum repo downloading error: Downloading error(s): repodata/dbae36491ad0ee21f407dfa338160598684f14fdfc1dfd62aa498f1ac7066e90-filelists.xml.gz - Cannot download, all mirrors were already tried without success
```

**Expected behavior**
Yum to complete without error

**Environment (please complete the following information):**
- OS: [Rocky Linux 8.6]
- Hardware: [64-bit]
- Themis version: [0.14.0-1]
- Installation way:
   - [x] via package manager
   - [ ] built from source

**Additional context**
Add any other relevant context for the problem here. Share an example project, if you can.

**Comment by vixentael** [comment]
hi @dev10! Sorry for delay. That's weird, taking a look

**Comment by dev10** [comment]
Hi @vixentael, the issue has still not been resolved unfortunately

**Comment by vixentael** [comment]
Yes, unfortunately. We have put it into the backlog, will do as soon as have fee hands.

cc @shadinua here

**Comment by dev10** [comment]
Ok, thanks for the update
